### PR TITLE
complete signup page

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
image

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,245 @@
+@import url("main.css");
+
+
+.nav-bar {
+    position: absolute;
```
Comment (JuvitaSaini09) [inline index.css]: ig this is of no use as u gave position at line number 10

File: index.css [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+    display: flex;
+    gap: 2rem;
+    align-items: center;
+    justify-content: center;
+    width: 50%;
+    padding: 1rem;
+}
+
+.nav--search--input {
+
```
Comment (JuvitaSaini09) [inline index.css]: remove space

File: index.css [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+    color: var(--WHITE);
+}
+
+.btn-primary {
+    background-color: gray;
+}
+
+
+.bg-img {
+
```
Comment (JuvitaSaini09) [inline index.css]: remove space

File: index.css [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+}
+
+.signup-title {
+    padding: 0rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 48px;
```
Comment (JuvitaSaini09) [inline index.css]: u could have used rem here

File: index.css [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+    line-height: 65px;
```
Comment (JuvitaSaini09) [inline index.css]: use rem

File: index.html [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+    <title>QUANTUM UI | Home</title>
+</head>
+
+<body>
+
+    <!--navigation--container -->
+
+
+    <div class="nav-bar">
+        <div class="nav--body">
```
Comment (JuvitaSaini09) [inline index.html:23]: just a suggestion ,you can also use nav tag for semantic html

File: index.html [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+    </div>
+
+
+
+
+
+    <!--main--container -->
+
+    <div class="main--container">
+        <img class="bg-img" src="/assets/francesco-gallarotti-ruQHpukrN7c-unsplash.jpg" alt="">
```
Comment (JuvitaSaini09) [inline index.html]: why is ur alt tag empty

**Review by JuvitaSaini09 — APPROVED** [review]
your code looks clean,i have suggested small changes

File: index.css [hunk 8]
```diff
@@ -0,0 +1,242 @@
...
+}
+
+.signup-title {
+    padding: 0rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 48px;
```
Comment (theVivek-kumar) [inline index.css]: use rem

**Review by theVivek-kumar — CHANGES_REQUESTED** [review]

**Pull request merged**